apps/vscode-wing/package.json

```
@@ -162,7 +205,19 @@
           }
         }
       }
-    ]
+    ],
+    "views": {
+      "explorer": [
+        {
+          "id": "consoleExplorer",
+          "name": "Wing Explorer"
```

Since this view has the resources, maybe this?

```suggestion
          "name": "Resource Explorer"
```

or

```suggestion
          "name": "Wing Resources"
```